packages/cxl-ui/src/components/cxl-icon-nav-item.js

```diff
@@ -0,0 +1,44 @@
+// Import the LitElement base class and html helper function
+import { LitElement, html } from 'lit-element';
+import { ifDefined } from 'lit-html/directives/if-defined';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import '@vaadin/vaadin-item';
+import cxlThemeStyles from '../styles/cxl-theme-css.js';
+import cxlIconNavItemStyles from '../styles/cxl-icon-nav-item-css.js';
+
+class CXLIconNavItemElement extends LitElement {
+  static get properties() {
+    return {
+      href: {
+        type: String
+      }
+    };
+  }
+
+  static get styles() {
+    return [cxlThemeStyles, cxlIconNavItemStyles];
+  }
+
+  render() {
+    return html`
+      <a href=${ifDefined(this.href)}>
+        ${this.navItemContent}
+      </a>
```

Comment:
We will need an optional clarity `<label>` strategy for icons, because people are really bad at remembering what icons mean.

We currently render `title` attribute already, can use it: `<cxl-icon-nav-item title="Active Training">cxl:nav-trainings</cxl-icon-nav-item>`

```html
<a href>
  ${this.navItemContent}
  <label>${this.title}</label>
</a>
```